pkg/route/route_test.go

```diff
@@ -0,0 +1,206 @@
+package route
```

Comment:
Since I don't see tests elsewhere in the repo - do we now add tests to the infra repo?